### PR TITLE
Updated reference from Superblocks Lab to Ethereum Studio

### PR DESCRIPTION
**Pull request by javier-tarazaga**
## Description

Updated the reference from Superblocks Lab to the new official Ethereum Studio IDE instead.

### CODE REVIEW

**Comment by jjmstark** [comment]
Should this be first in the list? I can see reasons both for and against, but I think we should think about that first. A sketch:

- It is something we built, can improve, and have high confidence in
- However, is the typical user of this page looking for a "beginner" IDE, or a more "pro" tool? Does it make sense to lead wth this?
- Is there a perception issue with favouring an official "ethereum.org" collaboration?

File: docs/developers/index.md [hunk 1]
```diff
@@ -149,6 +149,12 @@ Ethereum has a large and growing number of tools to help developers build, test,
 
 ## Integrated Development Environments (IDEs)
 
+**Ethereum Studio -** ***Official Ethereum on-boarding IDE. Web-based IDE with a built in browser blockchain, MetaMask integration, transaction logger and many other features to help you getting started coding DApps as fast as possible. ***
```
Comment (samajammin) [inline docs/developers/index.md]: Remove the space before the final "***" for proper markdown formatting. Current display:
Image 2019-10-13 at 8 20 46 PM

**Comment by samajammin** [comment]
Given that our user persona research indicates that most visitors are relatively new to Ethereum, I'm fine with this being first in the list. I'm not sure how I feel about labeling it "Official Ethereum...". Is it "official"? What does "official" even mean? I propose cutting that & focusing on a description that explains it's ideal for getting started... e.g.

> Ethereum Studio - Web-based IDE ideal for new developers looking to experiment with smart contracts. Built in browser blockchain, MetaMask integration, transaction logger and many other features to help you getting started coding DApps as fast as possible.

Another term I fear may cause confusion is "browser blockchain" - any thoughts on a clearer explanation?

**Comment by jjmstark** [comment]
That makes sense to me. Addressing your comment above, how about this description:

>Ethereum Studio - Web-based IDE ideal for new developers looking to experiment with smart contracts. Ethereum Studio features multiple templates, MetaMask integration, transaction logger, and a simulated in-browser blockchain to help you get started building on Ethereum as fast as possible.

**Comment by samajammin** [comment]
@javier-tarazaga nudge 😄

**Comment by javier-tarazaga** [comment]
@jjmstark @samajammin What about this as a description? 

```
Web-based IDE ideal for new developers looking to experiment with smart contracts. Ethereum Studio features multiple templates, MetaMask integration, transaction logger, and a built in-browser Ethereum Virtual Machine (EVM) to help you get started building on Ethereum as fast as possible.
```

**Pull request merged**